src/main/target/STM32_UNIFIED/target.h

```diff
@@ -189,7 +189,6 @@
 #define USE_UART6
 #define USE_UART7
 #define USE_UART8
-#define USE_LPUART1
```

Comment:
If we are removing this until we find a better solution, maybe we should just comment it out for now since there actually is a LPUART1 on the H7?